config/default.yml

```diff
@@ -636,6 +636,16 @@ Style/StringLiteralsInInterpolation:
     - single_quotes
     - double_quotes
 
+Style/StringMethods:
```

Comment:
I found it in `disabled.yml` and that's because we expect it to generate a fair amount of false positives. `StringMethods` might be OK to enable by default since the method `intern` is less likely to appear in other classes, as far as I can see.